dist/alt-with-runtime.js

```diff
@@ -44,6 +44,23 @@ var getInternalMethods = function (obj, excluded) {
   }, {});
 };
 
+var getInheritedMethods = function (obj) {
+  var methods = arguments[1] === undefined ? {} : arguments[1];
```

Comment:
I would do

`var getInheritedMethods = function(obj, methods={}) { ... }` instead of using `var methods = arguments[1] === undefined ? {} : arguments[1];`.